Assets/Tests/InputSystem.Editor/InputActionAssetManagerTests.cs

```diff
@@ -136,6 +139,20 @@ public void Editor_InputActionAssetManager_CanDeleteAssetOnDisk()
             Assert.Throws<Exception>(() => inputActionAssetManager.SaveChangesToAsset());
         }
     }
+
+    [Test]
+    [Category("Editor")]
+    public void Editor_InputActionAssetManager_ActionsCodeGeneration_TypeNamesAreNotAffectedByCultureChange()
```

Comment:
Awesome you added this test!

Reply:
Since this is a very slim test case targeting a previous regression maybe add [Description("See <url-to-issue>")] for future reference?

Reply:
Yes, good point. I didn't add the case because it's apparently a private one, as I've just noticed. That also makes my changelog entry incorrect. This case refers to a public case, but the public one is actually a UUM case. Hmm, I'll check with Paulius on my options here.

Reply:
Actually addressed this one.